content/en/blog/_posts/2019-12-21-Kubernetes-on-MIPS.md

```diff
@@ -0,0 +1,144 @@
+| etcd for CKE on MIPS           | 3.2.26  | `registry.inspurcloud.cn/library/cke/etcd/etcd-mips64el:v3.2.26`                                                                                                                                                                                                                  |
+| pause for CKE on MIPS          | 3.1     | `registry.inspurcloud.cn/library/cke/kubernetes/pause-mips64el:3.1`                                                                                                                                                                                                               |
+| hyperkube for CKE on MIPS      | 1.14.3  | `registry.inspurcloud.cn/library/cke/kubernetes/hyperkube-mips64el:v1.14.3`                                                                                                                                                                                                       |
+| coredns for CKE on MIPS         | 1.6.5   | `registry.inspurcloud.cn/library/cke/kubernetes/coredns-mips64el:v1.6.5`                                                                                                                                                                                                          |
+| calico for CKE on MIPS         | 3.8.0   | `registry.inspurcloud.cn/library/cke/calico/cni-mips64el:v3.8.0` `registry.inspurcloud.cn/library/cke/calico/ctl-mips64el:v3.8.0` `registry.inspurcloud.cn/library/cke/calico/node-mips64el:v3.8.0` `registry.inspurcloud.cn/library/cke/calico/kube-controllers-mips64el:v3.8.0` |
+
+> Note: CKE is a kubernetes based cloud container engine launched by Inspur
+
+![K8S-MIPS Cluster Components](/images/blog/2019-12-21-Kubernetes-on-MIPS/k8s-mips-cluster-components.png)
+
+*Figure 2 K8S-MIPS Cluster Components*
+
+![CPU Architecture](/images/blog/2019-12-21-Kubernetes-on-MIPS/cpu-architecture.png)
+
+*Figure 3 CPU Architecture*
+
+![Cluster Node Information](/images/blog/2019-12-21-Kubernetes-on-MIPS/cluster-node-information.png)
+
+*Figure 4 Cluster Node Information*
+
+## Run K8S Conformance Test
+
+The most straightforward way to verify the stability and availability of the K8S-MIPS cluster is to run Kubernetes [conformance test](https://github.com/kubernetes/kubernetes/blob/v1.16.2/cluster/images/conformance/README.md).
+
+Conformance is a standalone container to launch Kubernetes end-to-end tests, for the purposes of conformance testing.
+
+Once the test started, it will launches a number of pods for various end-to-end tests. The source code of those images used by these pods is mostly from `kubernetes/test/images` , and the built images are pulled from repository `gcr.io/kubernetes-e2e-test-images/` . Since there are no MIPS images in the repository, we must first build all needed images to run the test.
```

Comment:
```suggestion
Once the test has started, it will launches a number of pods for various end-to-end tests. The source code of those images used by these pods is mostly from `kubernetes/test/images` , and the built images are pulled from repository `gcr.io/kubernetes-e2e-test-images/` . Since there are no MIPS images in the repository, we must first build all needed images to run the test.
```